src/core/Header.tsx

```diff
@@ -91,6 +100,15 @@ export type HeaderProps = {
            */
           href: string;
         };
+        /**
+         * Upgrade link information.
+         */
+        upgrade: {
```

Comment:
So, I don't think this field actually exists in `sessionState`, at least according to what I can see is surfaced [elsewhere](https://github.com/ably/website/blob/main/app/helpers/application_helper.rb#L427-L435). 

Is there a plan to add it?

It's probably possible to piece it together from the user id here, but to match convention it would need to be surfaced from the `website` end.